modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementExamples.java

```diff
@@ -90,6 +108,10 @@ protected IamPolicyManagementExamples() { }
   private static String exampleV2PolicyEtag = null;
   private static String exampleCustomRoleId = null;
   private static String exampleCustomRoleEtag = null;
+  private static String exampleTemplateId = null;
```

Comment:
Same comments here as in the Go PR about inter-operation dependencies, links, etc.